src/IdTokenResponse.php

```diff
@@ -21,15 +21,18 @@ class IdTokenResponse extends BearerTokenResponse
     protected ClaimExtractor $claimExtractor;
 
     private Configuration $config;
+    private ?string $issuer;
 
     public function __construct(
```

Comment:
I think we can use constructor property promotion in PHP 8.0. So that we can combine class fields, constructor definition and variable assignments all into one syntax, in the construct parameter list.

Reply:
The package is still compatible with PHP 7.4 (that's what is declared in `composer.json`) so I guess this is a no go for this version at least.